docs/topic-guides/users.txt

```diff
@@ -3,52 +3,53 @@
-create a Determined *user* account for each developer. After authenticating as
-a given user, the Determined objects created in that session will be
-associated with that user account. By default, the WebUI and CLI will
-only display objects created by the current user.
+Determined is designed for teams of machine learning developers. Objects, such as experiments
+and notebooks, created by a developer will be visible to all users of the system.
+
+
+Getting Started
+---------------
+
+Each Determined installation comes with the two user accounts ``determined`` and ``admin``.
 
-Initially, a Determined installation has two users: ``admin`` and ``determined``.
-The ``determined`` user is the default account that is used for WebUI and CLI
-operations. If individual user accounts are not required, no additional
-configuration is needed. In this case, all assets created within Determined
-(experiments, commands, etc.) will be attributed to ``determined``.
+The ``determined`` user is the default account that is used for CLI operations.
+If individual user accounts are not required, users can log in as the user
+``determined`` and no additional configuration is needed. In this case, all assets
+created within Determined (experiments, commands, etc.) will be attributed to
+``determined``.
 
 The ``admin`` user can be used to create users, change other users'
 passwords, and activate/deactivate users.
 
-.. note::
+.. warning::
 
-   The ``admin`` user initially has a blank password. The ``user
-   change-password`` CLI command can be used by the admin to update this
-   password:
+   The ``admin`` and ``determined`` users initially have blank passwords.
 
-   .. code::
 
-      det -u admin user change-password
+Use the ``det user change-password`` to set a password for the ``admin`` user.
 
-Getting Started
----------------
+.. code::
 
-To start using users in a Determined cluster, the system administrator should
-first set a password for the ``admin`` user as described above.
+   det -u admin user change-password
 
-Next, the ``admin`` user should deactivate the ``determined`` user:
+We also suggest deactivating the user the user ``determined``:
 
-::
+.. code::
 
    det -u admin user deactivate determined
 
 This ensures that no one can access the Determined cluster as the default user
-(any objects that were created by this user will remain). Finally, the
-system administrator should create user accounts for each individual who
-would like to access Determined:
+(any objects that were created by this user will remain).
+
+While using the existing ``determined`` account is convenient for small teams, we encourage
+everyone, especially larger teams, to create a Determined user account for each developer.
+After authenticating as a given user, the Determined objects created in that session will be
+associated with that user account. By default, the WebUI and CLI will
+only display objects created by the current user.
```

Comment:
blocking: I would move this to part of the ``determined`` user explanation.

Reply:
Would make the wording more concrete too: "Teams will benefit from having clearer authorship and improved filtering by creating a Determined user account for each individual. Creating users can be done as ``admin`` using the following command:"

I don't think we need to explain more than that, since the rest of the behavior is pretty common for apps that have users.